pepys_import/core/store/data_store.py

```diff
@@ -2063,3 +2058,57 @@ def edit_items(self, items, edit_dict):
                     previous_value=str(getattr(item, col_name)),
                     change_id=change_id,
                 )
+
+    def find_dependent_objects(self, table_name: str, id_list: list) -> dict:
```

Comment:
This method takes a table_name parameter as a string, but the `delete_objects` method takes a table object instead. Could you make them match? Ideally, by making both functions take either table names or objects using the:

```
if isinstance(table_obj, str):
    table_obj = self._get_table_object(table_obj)
```

Reply:
I changed table_obj to table_name: https://github.com/debrief/pepys-import/pull/789/commits/c766544a7abc49a54b98e8d8a355684d8ae850d1